skimage/transform/setup.py

```diff
@@ -30,12 +34,13 @@ def configuration(parent_package='', top_path=None):
     return config
 
 if __name__ == '__main__':
-    from numpy.distutils.core import setup
+    from numpy.distutils.core import setup, Extension
     setup(maintainer='scikit-image Developers',
           author='scikit-image Developers',
           maintainer_email='scikit-image@googlegroups.com',
           description='Transforms',
           url='https://github.com/scikit-image/scikit-image',
           license='SciPy License (BSD Style)',
+          ext_modules=[Extension(extra_compile_args=['/EHsc']),],
```

Comment:
What is this line doing? I searched for this motif and couldn't find much except for [this line in pyzmq](https://github.com/zeromq/pyzmq/blob/master/setup.py#L524). The CI builds are happy, I'm just asking because it affects how the entire package gets built.